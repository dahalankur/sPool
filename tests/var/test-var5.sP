# variable names can be very long
qwertyuiopjhjmnbvcxzsdfdsawerewq1234567890987654________123456YTREWQWESDXCVBNMYTFDSDFCV12345fjdklsajfdkslafjdsfljskdewuriourqwieorpuewqiropquewirpoqwuriojfdksl______eggs_and_bacon______1bologna
