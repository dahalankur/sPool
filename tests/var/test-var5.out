PROGRAM {[ 
    VAR(qwertyuiopjhjmnbvcxzsdfdsawerewq1234567890987654________123456YTREWQWESDXCVBNMYTFDSDFCV12345fjdklsajfdkslafjdsfljskdewuriourqwieorpuewqiropquewirpoqwuriojfdksl______eggs_and_bacon______1bologna)]}
